marketo_ma.install

```diff
@@ -5,7 +5,68 @@
  * Install hooks for Marketo MA module.
  */
 
-use Drupal\Core\Link;
+/**
+ * Implements hook_schema().
```

Comment:
Thanks @Jaesin. I'm okay with changing the implementation back to something more D8 appropriate. What we had before was resulting in a lot of hits to the describe API and needed more persistence.